docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
       - ${DOCKER_RABBITMQ_CONSOLE_PORT-15672}:15672
 
   minio:
-    image: minio/minio:latest
+    image: minio/minio:2022.3.3
```

Comment:
Is this a valid tag for the official non-bitnami minio docker image? I briefly glanced at https://hub.docker.com/r/minio/minio/tags and they seem to use a different versioning scheme.

Reply:
Good catch, that was lazy on my part. Seems like `minio/minio:RELEASE.2022-03-03T21-21-16Z` is the corresponding version. I'll update it.